test/cmdlineTests/function_debug_info_via_yul/output

```diff
@@ -0,0 +1,11 @@
+{
+  "contracts":
+  {
+    "function_debug_info_via_yul/input.sol:C":
+    {
+      "function-debug": {},
+      "function-debug-runtime": {}
```

Comment:
So it does not work for EVM bytecode generated from Yul? What do we need to make this happen?

Reply:
Exactly. We need to extend the debug info data structure that @mijovic is currently working on and pull it through using annotations.